src/omv/modules/py_tf.c

```diff
@@ -578,9 +483,11 @@ STATIC void py_tf_segment_output_callback(void *callback_data,
 }
 
 STATIC mp_obj_t py_tf_segment(uint n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
-    enum { ARG_roi };
+    enum { ARG_roi, ARG_input_scale, ARG_input_normalization };
     static const mp_arg_t allowed_args[] = {
-        { MP_QSTR_roi, MP_ARG_OBJ | MP_ARG_KW_ONLY, {.u_rom_obj = MP_ROM_NONE} }
+        { MP_QSTR_roi, MP_ARG_OBJ | MP_ARG_KW_ONLY, {.u_rom_obj = MP_ROM_NONE} },
+        { MP_QSTR_input_scale, MP_ARG_INT | MP_ARG_KW_ONLY, {.u_int = PY_TF_SCALE_0_1} },
+        { MP_QSTR_input_normalization, MP_ARG_OBJ | MP_ARG_KW_ONLY, {.u_rom_obj = MP_ROM_NONE} },
```

Comment:
```suggestion
        { MP_QSTR_input_stats, MP_ARG_OBJ | MP_ARG_KW_ONLY, {.u_rom_obj = MP_ROM_NONE} },
```

Reply:
This is an outdated comment. I changed it previously to what was discussed in the meeting with mean/std.